### PR TITLE
COG-221 CALS-admin role authorized to read and edit users index

### PR DESCRIPTION
**Pull request by sorych**
### JIRA Issue Link
- [COG-221: CALS-admin role authorized to read and edit users index](https://osi-cwds.atlassian.net/browse/COG-221)

### Technical Description
CALS-admin role authorized to read and edit users index

### Tests
- [x] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 
Please indicate why tests were not added.

### Types of changes
What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:
Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by sorych** [comment]
@dmytro-palczewski-osi please review

File: docker-es-xpack/xpack_policies.json [hunk 1]
```diff
@@ -325,5 +325,30 @@
...
+    "action": "POST /_xpack/security/role/CALS-admin",
+    "content": {
+      "cluster": [],
+      "indices": [
+        {
+          "names": [
+            "users"
+          ],
+          "privileges": [
+            "read", "write"
```
Comment (dmytro-palczewski-osi) [inline docker-es-xpack/xpack_policies.json]: Currently only getUser() operation is accessible for Cals-admin so write privilege probably has to be removed?

File: docker-es-xpack/xpack_policies.json [hunk 2]
```diff
@@ -325,5 +325,30 @@
...
+          ],
+          "query": {
+            "template": {
+              "inline": {
+                "match": {
+                  "roles": "CALS-external-worker"
+                }
```
Comment (dmytro-palczewski-osi) [inline docker-es-xpack/xpack_policies.json]: or CALS-admin?

Comment (sorych) [reply docker-es-xpack/xpack_policies.json]: the requirement says "Default list of users for CALS-ADMINS should be those users that are marked with user role of CALS-external-worker"

**Pull request merged**